src/main/scala/edu/berkeley/ce/rockslicing/InputProcessor.scala

```diff
@@ -1,16 +1,15 @@
 package edu.berkeley.ce.rockslicing
 
-import breeze.linalg.DenseVector
-
 import scala.io.Source
 import scala.util.Try
 
 object InputProcessor {
   // Processes input file: Add rock volume faces and joints to respective input list
-  def readInput(inputSource: Source): Option[(Array[Double],
-                                             Seq[Face], Seq[Joint])] = {
+  def readInput(inputSource: Source): Option[ (Array[Double], Array[Double],
+    Seq[Array[Double]], Seq[JointSet]) ] = {
```

Comment:
I'll add a case class for this. `Face` won't work since the inputs also contain a point in the plane that defines the face - this isn't stored in `Face` since everything describing the face is relative to some local origin.